salt/template.py

```diff
@@ -64,14 +71,16 @@ def compile_template(template,
         input_data = ifile.read()
         if not input_data.strip():
             # Template is nothing but whitespace
-            return {}
+            log.error('Template is nothing but whitespace: {0}'.format(template))
+            return ret
 
     input_data = string_io(input_data)
     for render, argline in render_pipe:
         try:
             input_data.seek(0)
-        except Exception:
-            pass
+        except Exception as exp:
+            log.error('error: {0}:\n{1}'.format(exp))
```

Comment:
This breaks the renderer in certain cases because the format call is not properly constructed. I have removed the ``{{1}}`` since it appears to be unused.